### ISSUE TITLE
[L5] Produce L1A CDF formatted data products for IMAP-Hi

### ISSUE THREAD

**Issue by tech3371** [IMAP-Science-Operations-Center/imap_processing#124]
### Summary of the L5 requirement

The SDC needs to produce L1A CDF formatted data products for the IMAP-Hi instrument. The data products for L1A can be found in the IMAP-Lo algorithm document here: https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+Algorithm+Document+from+Instrument+Teams

### Parent requirement

SOC-SDC-L4-29 | SDC Execution of L1 Processing Software | The SDC shall be capable of executing L1 processing software to produce L1 data products within seven (7) days of receipt of all requisite data inputs. | Requirement | The SDC produces L1 data products via Instrument Team supplied algorithms and SDC produced processing code. | SOC-L3-57 | Test | SIT-3: L0 -> L1 processing | SIT-3
-- | -- | -- | -- | -- | -- | -- | -- | --

### Tasks

**Tasks**
- [x] #189
- [x] #190
- [x] #191

**Comment by tmplummer** [comment]
@tech3371 @sdhoyt , Are these tickets with the `Req: L5` labels being used to track anything? These are from before I was on the project and I don't know if these are something that I can cleanup/close or not.

**Comment by sdhoyt** [comment]
@subagonsouth No, all the L5 issues can probably be deleted unless you were specifically using it. This was my failed attempt to decompose the level 4 requirements into more testable requirements and track them in Github and we've moved away from doing this. Sounds like you're not using this, so I'll close it with this comment, but feel free to reopen if you are.
